src/website/src/app/documentation/demos/spinners/spinner-component.html

```diff
@@ -0,0 +1,39 @@
+<!--
+  ~ Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+  ~ This software is released under MIT license.
+  ~ The full license information can be found in LICENSE in the root directory of this project.
+  -->
+
+<h3 id="guidelines">Spinner Component</h3>
+<p>
+    Clarity provide component <code class="clr-code">clrSpinner</code>. 
+</p>
+
+<div class="clr-example squeeze">
+    <clr-spinner>Loading data ...</clr-spinner>
+</div>
+<clr-code-snippet [clrCode]="example"></clr-code-snippet>
+
+<h5>Modify</h5>
+<p>
+    Add modifier like <code class="clr-code">inline</code> or <code class="clr-code">inverse</code>. If you need to make 
+    change on the size you could add <code class="clr-code">small</code> or <code class="clr-code">medium</code>, the 
+    default size is auto set to <code class="clr-code">large</code>.
+</p>
+
+<div class="clr-example squeeze">
+    <clr-spinner inline>Downloading...</clr-spinner>
```

Comment:
Ellipsis.

Also, these should use an ngIf and a button to turn them on, otherwise they'll announce first thing when someone opens the page. This is probably good to mention in the docs here. The real value is the live region, so we need to make it clear _why_ this is used and to always put ngIf on it.